docs/packages.md

```diff
@@ -105,5 +112,10 @@ allowed)
 #### Publishing on espanso hub
 
 After following all these steps, you can request to publish your package to [espanso hub](http://hub.espanso.org)
-by opening an [Issue](https://github.com/federico-terzi/espanso-hub/issues) with the following information:
-* The package **repository** url
\ No newline at end of file
+by opening an [Issue](https://github.com/federico-terzi/espanso-hub/issues) with the following format:
```

Comment:
Could you please revert this part? These changes are not needed as we are going to refactor the package system in the near future

Reply:
Isn't it better to change it after refactoring?

Reply:
@federico-terzi ?

Reply:
@MuhammedZakir at the moment, I'm not sure how near that future will be :) So better to keep this part as it is for now